atomsci/ddm/utils/datastore_functions.py

```diff
@@ -2018,13 +2025,13 @@ def bulk_update_kv(file, client=None, i=0):
         dataset_key=row[1]
         kv_add=row[2]
         print('i =', i, 'dataset_key = ', dataset_key)
-        if type(kv_add) == str:
+        if isinstance(kv_add, str):
```

Comment:
test?

Reply:
This will need to wait for a later release when the datastore is back up.